### PR TITLE
Release 10.2.0

### PR DESCRIPTION
**Pull request by r0mant**
Do not merge yet, a couple PRs still need to be merged.

### CODE REVIEW

**Review by russjones — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,26 @@
 # Changelog
 
+## 10.2.0
+
+This release of Teleport contains multiple improvements and bug fixes.
+
+* Fixed issue with `tsh` reporting "no suitable devices found" when using Yubikeys. [#16011](https://github.com/gravitational/teleport/pull/16011)
+* Fixed potential panic when using unsupported Google application credentials file. [#16042](https://github.com/gravitational/teleport/pull/16042)
+* Fixed issue with database connections getting terminated due to idle timeout when running long queries. [#16017](https://github.com/gravitational/teleport/pull/16017)
+* Fixed issue with shared directory operations terminating Desktop Access sessions. [#16054](https://github.com/gravitational/teleport/pull/16054)
```
Comment (zmb3) [inline CHANGELOG.md:10]: I think we can remove this one. Directory sharing is new to this release, so any issues that it fixed were only present on master.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,26 @@
...
+* Fixed issue with IAM joining in AWS China regions. [#15915](https://github.com/gravitational/teleport/pull/15915)
```
Comment (zmb3) [inline CHANGELOG.md:11]:
```suggestion
* Add support for IAM joining in AWS China regions. [#15915](https://github.com/gravitational/teleport/pull/15915)
```

This is more of a new feature than a bug fix, right?

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,26 @@
...
+* Fixed issue with AWS Console Access in AWS GovCloud regions. [#16067](https://github.com/gravitational/teleport/pull/16067)
+* Fixed issue with `tsh ssh` and `tsh aws` commands failing with "unknown flag" errors. [#16094](https://github.com/gravitational/teleport/pull/16094)
+* Fixed issue with lock target not being included in `lock.create` audit events. [#15981](https://github.com/gravitational/teleport/pull/15981)
```
Comment (zmb3) [inline CHANGELOG.md:14]:
```suggestion
* Added the lock target to `lock.create` audit events. [#15981](https://github.com/gravitational/teleport/pull/15981)
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,26 @@
...
+* Fixed issue with empty Github connector fields being always marshaled. [#16012](https://github.com/gravitational/teleport/pull/16012)
+* Fixed issue with SSH sessions not properly terminating in some cases. [#16065](https://github.com/gravitational/teleport/pull/16065)
+* Introduced a new web UI guided wizard for joining SSH nodes and Windows desktops. [#16169](https://github.com/gravitational/teleport/pull/16169), [#16087](https://github.com/gravitational/teleport/pull/16087)* Added support for Azure PostgreSQL and MySQL databases auto-discovery. [#15988](https://github.com/gravitational/teleport/pull/15988), [#15990](https://github.com/gravitational/teleport/pull/15990), [#15989](https://github.com/gravitational/teleport/pull/15989), [#15991](https://github.com/gravitational/teleport/pull/15991), [#15992](https://github.com/gravitational/teleport/pull/15992)
```
Comment (zmb3) [inline CHANGELOG.md:17]:
```suggestion
* Introduced a new web UI guided wizard for joining SSH nodes. [#16169](https://github.com/gravitational/teleport/pull/16169), [#16087](https://github.com/gravitational/teleport/pull/16087)* Added support for Azure PostgreSQL and MySQL databases auto-discovery. [#15988](https://github.com/gravitational/teleport/pull/15988), [#15990](https://github.com/gravitational/teleport/pull/15990), [#15989](https://github.com/gravitational/teleport/pull/15989), [#15991](https://github.com/gravitational/teleport/pull/15991), [#15992](https://github.com/gravitational/teleport/pull/15992)
```

Windows Desktops is going to slip to 10.2.1 - let's remove.

File: CHANGELOG.md (hunk 1 above)
Comment (zmb3) [reply CHANGELOG.md:10]: Instead of removing it, let's just say "Add support for directory sharing to desktop access."

**Review by hatched — APPROVED** [review]

**Pull request merged**